tools/distpartitioning/convert_partition.py

```diff
@@ -9,7 +9,9 @@
 import pandas as pd
 import constants
 from pyarrow import csv
-from utils import read_json, get_idranges
+from utils import read_json, get_idranges, mem_snapshot
+
+import gc
```

Comment:
Use isort to keep the import in order?
https://quip-amazon.com/J9cMAtI9NOuC/Python-Code-Style-Check#temp:C:KXU4172bb625aae4bb088505bd9e

Reply:
Done as suggested. will be included in a separate commit